src/patchwork/components/SpatialSidebar.tsx

```diff
@@ -532,27 +506,27 @@ const DiscusssionCommentView = ({
   );
 };
 
-const AnnotationView = <T, V>({
+const AnnotationsView = <T, V>({
   docType,
-  annotation,
+  annotations,
 }: {
   docType: DocType;
-  annotation: Annotation<T, V>;
+  annotations: Annotation<T, V>[];
 }) => {
   switch (docType) {
```

Comment:
Can we replace this switch statement by defining a mapping from datatypes to annotation viewers, just like we use for regular full-doc viewers?